### PR TITLE
[RFR] Containers Templates Test

### PR DESCRIPTION
**Pull request by pavelzag**
{{pytest: cfme/tests/containers/test_containers_templates.py -v --use-provider cm-env1 }}

Purpose or Intent
=================
This test checks the Container Templates menu.
This PR contains a Container Template object for additional tests

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3c160ffdc00c72e09f59bf1420fa2cf507bf615d:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by psav** [comment]
I believe this is failing because Container Templates is concatenated in the menu :(

**Comment by pavelzag** [comment]
Truncated because of the resolution? Can't access the Report and the Artifacts on PRT for some reason..  Running fine here on a bigger screen I guess

**Comment by pavelzag** [comment]
Yea, it failed because it couldn't add the provider (The environment is still down, we have some major issues there)

**Comment by psav** [comment]
Moving back to WIPTEST as PRT didn't check due to bad env

File: cfme/tests/containers/test_containers_templates.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+from cfme.web_ui import CheckboxTable
+from utils import testgen
+from utils.version import current_version
+from utils.appliance.implementations.ui import navigate_to
+from cfme.containers.template import Template
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
```
Comment (otsuman) [inline cfme/tests/containers/test_containers_templates.py]: we do not support versions below 5.6, so no need for this

File: cfme/tests/containers/test_containers_templates.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+@pytest.mark.parametrize('rel', TEMPLATE_RELATIONSHIP_FIELDS)
+def test_containers_templates_relationships(template, rel):
+    """ This test checks the Relationship fields of Container Templates
+        under Container Providers
+       Steps:
+           * Navigate to Compute -> Containers -> Container Templates.
+           Verify there's a list of Templates
+           * Select a random Container Template and check its' Relationships fields
+       """
+    assert template.get_detail('Relationships', rel)
```
Comment (otsuman) [inline cfme/tests/containers/test_containers_templates.py]: Just checking if text in filed is present? What if value is incorrect?

Comment (pavelzag) [inline cfme/tests/containers/test_containers_templates.py]: According to test's definition in Polarion this means failure

**Comment by dajoRH** [comment]
Lint report for commit 6157c30d4b9882e7370daa5b7dc372d66c17996a:

`cfme/tests/containers/test_containers_templates.py`:
- :red_circle: Line 6:1: **F401** *'utils.version.current_version' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ebe9f112d838a29565cee5d18165ce0ee7bc5b78:

`cfme/tests/containers/test_containers_templates.py`:
- :red_circle: Line 6:1: **F401** *'utils.version.current_version' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/containers/test_containers_templates.py (hunk 2 above)
Comment (pavelzag) [inline cfme/tests/containers/test_containers_templates.py]: @otsuman Hey, please review again, thanks!

Comment (psav) [inline cfme/tests/containers/test_containers_templates.py]: @pavelzag the shouldn't the test check it matches instead of just checking existence?

Comment (pavelzag) [inline cfme/tests/containers/test_containers_templates.py]: At the moment no, at least according to Polarion's definition it shouldn't. But I will add data integrity as next step.

**Pull request merged**